### PR TITLE
tracing: Move tracing test to different job

### LINKED ISSUE #4195: Tracing test no longer running
Since the tracing test was previously run in the CRI_CONTAINERD_K8S_MINIMAL (jenkins-ci-ubuntu-1804-containerd-k8s-e2e-minimal) and it was disabled, we have the following options:
1. Move the tracing test to CRI_CONTAINERD_K8S (jenkins-ci-ubuntu-20-04)
2. Repurpose/rename CRI_CONTAINERD_K8S_MINIMAL for tracing test (and future tests) and add myself as an owner (See [403](https://github.com/kata-containers/ci/issues/403))

I think 2 is probably best since CRI_CONTAINERD_K8S does a lot already. For now I need to test 1 for agent-shutdown-test debugging.

cc @jodh-intel @GabyCT

### PR DESCRIPTION
**Pull request by cmaf**
Since the current job that ran the tracing test is disabled, move it to
an existing job.

Fixes #4195

Signed-off-by: Chelsea Mafrica <chelsea.e.mafrica@intel.com>

### CODE REVIEW

**Comment by cmaf** [comment]
@jodh-intel do you know why this would be happening? It is causing the job to fail.
```
+ cd /tmp/jenkins/workspace/kata-containers-2.0-tests-ubuntu20.04-PR/go/src/github.com/kata-containers/kata-containers/src/trace-forwarder
tracing/test-agent-shutdown.sh: line 1270: cd: /tmp/jenkins/workspace/kata-containers-2.0-tests-ubuntu20.04-PR/go/src/github.com/kata-containers/kata-containers/src/trace-forwarder: No such file or directory
```

**Comment by cmaf** [comment]
Some notes:

The current tracing/agent shutdown tests on jenkins-ci-ubuntu-20-04 add ~7 minutes to the job; running independently was about 22 minutes.

**Review by jodh-intel — APPROVED** [review]
Thanks @cmaf.

lgtm

**Comment by jodh-intel** [comment]
We need 1 more ack here.

**Comment by GabyCT** [comment]
@jodh-intel this is still having some issues with the agent test
```
03:43:03 + info 'OCI bundle directory : /tmp/tmp.YLgnOTB3Pc'
03:43:03 + local 'msg=OCI bundle directory : /tmp/tmp.YLgnOTB3Pc'
03:43:03 ++ basename tracing/test-agent-shutdown.sh
03:43:03 + echo -e '[test-agent-shutdown.sh:604] INFO: OCI bundle directory : /tmp/tmp.YLgnOTB3Pc'
03:43:03 [test-agent-shutdown.sh:604] INFO: OCI bundle directory : /tmp/tmp.YLgnOTB3Pc
03:43:03 + return 0
03:43:03 make: *** [Makefile:95: agent-shutdown] Error 1
03:43:03 [run.sh:58] ERROR: sudo -E PATH=/tmp/jenkins/workspace/kata-containers-2.0-tests-ubuntu20.04-PR/go/bin:/usr/local/go/bin:/usr/sbin:/sbin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin bash -c make tracing
03:43:03 [jenkins_job_build.sh:215] ERROR: .ci/run.sh
03:43:03 Build step 'Execute shell' marked build as failure
```
so far we do not have any of the tracing tests running in any CI

**Comment by jodh-intel** [comment]
@GabyCT - I've added the dnm label and will try to find some time to debug...

**Comment by fidencio** [comment]
A head's up that may affect your PR.

In the Architecture Committee meeting from January 25th, 2022, the Architecture Committee has agreed on using the "Dismiss stale pull request approvals when new commits are pushed" configuration from GitHub.  It basically means that if your PR has been rebased or updated, the approvals given will be erased.

In order to minimize traumas due to the new approach, please, consider adding a note on the changes done before the rebase / force-push, and also pinging the reviewers for a subsequent round of reviews.

Thanks for your understanding!

Related issue: https://github.com/kata-containers/community/issues/249

**Comment by GabyCT** [comment]
@cmaf can we drop the `do-not-merge` label for this PR?

**Comment by cmaf** [comment]
@GabyCT yes we can. What do you think about adding it to this job for now? I ran the test many times and haven't seen any failure. 

Also @jodh-intel could you PTAL again?

**Review by GabyCT — APPROVED** [review]

**Comment by GabyCT** [comment]
@cmaf could you disable please the test in `ppc64le` CI as it seems that has some issues (maybe related with the hw)
```
24187fb4c8755d4e70f859320c9f26b7b06ec1ea749cfc31efc7f2840c02b316
docker: Error response from daemon: driver failed programming external connectivity on endpoint jaeger (a3802e698aa8f3713fd4dcae02375a853d39f32a045200f49c415157c1d8f808):  (iptables failed: iptables --wait -t filter -A DOCKER ! -i docker0 -o docker0 -p tcp -d 172.17.0.2 --dport 16686 -j ACCEPT: iptables: No chain/target/match by that name.
```

**Review by devimc — APPROVED** [review]

**Pull request merged**
